### PR TITLE
Capstone Project Module 2 Javascript & API

### PR DESCRIPTION
**Pull request by sntgopl**
In this milestone, @Camilovelag and I did:

A dynamic render for the HTML homepage and popUp.
A fetch function for the information to display in the rendered content.
A function to create a new API id for comments and likes and add interaction.
Tests for all functions rendering the data in the webpage.
Counter functions to keep track of interactions and data.

### CODE REVIEW

File: src/modules/apis/getAppId.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const getAppId = async () => {
+  const response = fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+    method: 'POST',
+    // body: JSON.stringify(gameName),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const data = await (await response).text();
+  console.log(data); // eslint-disable-line
```
Comment (rustam-tolipov) [inline src/modules/apis/getAppId.js]: - [ ] Please kindly remove commented codes and unnecessary `console.log`s from your code 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+
+## Acknowledgements
+
+Microverse Community
+GitHub Repositories
+Minimalist Leaderboard list
+
+## 📝 License
+
+This project is MIT licensed.
```
Comment (rustam-tolipov) [inline README.md]: - [ ] [OPTIONAL] It would be great if you add a link to your license file [like this](URL)

File: src/modules/apis/getAppId.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const getAppId = async () => {
+  const response = fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+    method: 'POST',
+    // body: JSON.stringify(gameName),
```
Comment (rustam-tolipov) [inline src/modules/apis/getAppId.js]: - [ ] Kindly remove commented codes from your file.

File: src/modules/homepage/test/homepage.test.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    // crear html para anexarle con innerhtml
+    // seleccionar con queryselector
+    const section = document.querySelector('#homepage');
+    const counter = document.createElement('div');
+    counter.innerHTML = content;
+    section.appendChild(counter);
+    const number = document.querySelectorAll('.card');
+
+    // Assert
+    // const card = document.getElementsByTagName('section').length;
```
Comment (rustam-tolipov) [inline src/modules/homepage/test/homepage.test.js]: - [ ] To make your code quality perfect. Kindly remove commented codes from your file.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @sntgopl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @sntgopl, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 
congratulations


## Good Highlights✔

- Linter checks are passing
- Good commit message
- Adherence to GitHub flow
- Required changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**